codec-http/src/main/java/io/netty/handler/codec/http/Cookie.java

```diff
@@ -20,78 +20,76 @@
 /**
  * An interface defining an
  * <a href="http://en.wikipedia.org/wiki/HTTP_cookie">HTTP cookie</a>.
+ * @deprecated Use {@link io.netty.handler.codec.http.cookie.Cookie} instead.
  */
-public interface Cookie extends Comparable<Cookie> {
+@Deprecated
+public interface Cookie extends io.netty.handler.codec.http.cookie.Cookie {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this interface. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2176)